environment.yml

```diff
@@ -2,48 +2,57 @@ name: simpeg-test
 channels:
   - conda-forge
 dependencies:
+# dependencies
   - numpy>=1.20
   - scipy>=1.8
   - scikit-learn>=1.2
-  - pymatsolver>=0.2
-  - matplotlib
+  - pymatsolver-base>=0.2
+  - matplotlib-base
   - discretize>=0.10
   - geoana>=0.5.0
   - empymod>=2.0.0
-  - setuptools_scm
   - pandas
+
+# solver
+# uncomment the next line if you are on an intel platform
+#  - pydiso # if on intel pc
```

Comment:
Unfortunately there is not way to specify a conditional dependance in an `environment.yml` file. So this line is commented out with some notes to keep it platform independent.

Hopefully someone sees this before using the environment.yml file. #1356